pkg/question-nine/question.go

```diff
@@ -0,0 +1,23 @@
+package question_nine
+
+type listNode struct {
+	val  int
+	next *listNode
+}
+
+func reverseList(head *listNode) *listNode {
+	if head == nil {
+		return head
+	}
+	var prev *listNode = nil
+	cur := head
+	nxt := head.next
+	for nxt != nil {
+		cur.next = prev
+		prev = cur
+		cur = nxt
+		nxt = cur.next
+	}
+	cur.next = prev
+	return cur
+}
```

Comment:
Добавь конструктор и интерфейс у `listNode`
В конструктор по входным параметрам можно передавать  значение текущего элемента списка и ссылку на следующий элемент
```suggestion
    func NewListNode(...)
}
```